tests/unit/test_pack_unpack.py

```diff
@@ -0,0 +1,75 @@
+from collections import OrderedDict
+
+import pytest
+from authlib.common.encoding import json_dumps
+
+from didcomm.common.resolvers import ResolversConfig
+from didcomm.common.types import VerificationMethodType, VerificationMaterial, VerificationMaterialFormat
+from didcomm.message import MessageOptionalHeaders, MessageRequiredHeaders, MessageBody, Message
+from didcomm.pack_signed import pack_signed
+from didcomm.secrets.secrets_resolver import Secret
+from tests.common.example_resolvers import ExampleSecretsResolver, ExampleDIDResolver
+
+
+@pytest.mark.asyncio
+async def test_pack_signed():
+    alice_did = "did:example:alice"
+
+    alice_secret = Secret(
+        kid="did:example:alice#key-1",
+        type=VerificationMethodType.ED25519_VERIFICATION_KEY_2018,
+        verification_material=VerificationMaterial(
+            format=VerificationMaterialFormat.JWK,
+            value=json_dumps({
+                "kty": "OKP",
+                "d": "uuAE6HmqnCnVjkF0ygjZMQiHeYIvI3Qcwh_2SjGMG-o",
+                "crv": "Ed25519",
+                "x": "z0x6oKBZ-ehwn_tkBzbhav132eQ7vmj5s5Xen00rtW0"
+            })
+        )
+    )
+
+    secrets_resolver = ExampleSecretsResolver([alice_secret])
+
+    message = Message(
```

Comment:
not critical right now, but I believe we will have quite a lot of tests against out "test vectors", so it may make sense to put common values such as the Message into a helper method/fixture.

Doing this we can get much more compact and readable test, something like this:
```
@pytest.mark.asyncio
async def test_pack_signed(message, resolvers_config):
    result = await pack_signed(message, "did:example:alice", resolvers_config)
    expected_packed_msg = load_test_vector_signed_message()
    assert expected_packed_msg == result.packed_msg
    assert "did:example:alice#key-1" == result.sign_from_kid
```